File: src/main/java/org/antop/java/chapter4/LengthCounter.java
package org.antop.java.chapter4;

public class LengthCounter {
    private int counter = 0;

    public void addWord(String word) {
        counter += word.length();
    }

    public int getCounter() {
        return counter;
    }

    public static void main(String[] args) {
        LengthCounter lengthCounter = new LengthCounter();
        lengthCounter.addWord("hello");
        lengthCounter.addWord("antop");
        System.out.println(lengthCounter.getCounter());
    }
}

File: src/main/kotlin/org/antop/kotlin/chapter4/LengthCounter.kt
package org.antop.kotlin.chapter4

class LengthCounter {
    var counter: Int = 0
        private set

    fun addWord(word: String) {
        counter += word.length
    }
}

fun main() {
    val lengthCounter = LengthCounter()
    lengthCounter.addWord("hello")
    lengthCounter.addWord("antop")
    println(lengthCounter.counter) // 10
}
